omero/sysadmins/server-upgrade.txt

```diff
@@ -104,7 +104,21 @@ so that the app is available from the context menu on the webclient tree::
     $ bin/omero config append omero.web.open_with '["omero_figure", "new_figure",
       {"supported_objects":["images"], "target": "_blank", "label": "OMERO.figure"}]'
 
-
+Updating 'Open with' config
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default ``Open with`` configuration changed in OMERO 5.3.3 so that
+``Open with > Image Viewer`` *always* uses the old ``webgateway`` image viewer instead of
+using the viewer configured by :property:`omero.web.viewer.view`.
+However, the new default ``Open with`` configuration will be ignored if you have previously
+appended any new items to it, since the whole ``open_with`` list
+will then be stored in /etc/grid.xml.
+
+If you have configured :property:`omero.web.open_with` prior to OMERO 5.3.3 and
+you wish to have the old ``webgateway`` viewer available via ``Open with > Image Viewer``
```

Comment:
I just wanted to convey that this wasn't something new (I don't think users will recognise the "webgateway" name). Maybe "existing" or "standard" or "regular"? (don't want "default" as I've used that term elsewhere to refer to configs).